airbyte-integrations/connectors-performance/source-harness/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM airbyte/integration-base-java:dev AS build
+WORKDIR /airbyte
+ENV APPLICATION source-harness
+COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
+RUN tar xf ${APPLICATION}.tar --strip-components=1 && rm -rf ${APPLICATION}.tar
+FROM airbyte/integration-base-java:dev
+WORKDIR /airbyte
+ENV APPLICATION source-harness
+COPY --from=build /airbyte /airbyte
+COPY base.sh .
```

Comment:
Llooks like there is some duplication here from the integration-base-java image. Is that intentional?

Reply:
I rearranged the file, but logically this is doing something similar to what [source-postgres/Dockerfile](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-postgres/Dockerfile) is doing for example. 
Just adding a bunch of packages and its own base.sh